bdio-protobuf/src/main/proto/bdba_scan_nodes_v2.proto

```diff
@@ -88,6 +88,9 @@ message ProtoContainerNode {
 
   // list of real layer ids present in the given container, always has at least one value
   repeated string layers = 8;
+
+  // list of container paths, may be empty
+  repeated string image_paths = 9;
```

Comment:
Change this to be a list of file-entry ids instead of just path strings. We can then use files already emitted to the bdio with all the attributes there.

Reply:
Changed. I assume that if referenced file node is missing, bdio file is invalid.